client/styles/components/_sketch-list.scss

```diff
@@ -2,11 +2,75 @@
   overflow-y: auto;
   max-width: 100%;
   min-height: 100%;
+
+  @media (max-width: 770px) {
+    @include themify() {
+      background-color: getThemifyVariable("modal-background-color");
+    }
+
+    .sketches-table {
+      display: flex;
+      flex-direction: column;
+      padding: #{13 / $base-font-size}rem;
+      height: 100%;
+      overflow-y: auto;
+
+      thead {
+        display: none;
+      }
+
+      tbody {
+        display: flex;
+        flex-direction: column;
+        gap: #{12 / $base-font-size}rem;
+
+        .sketches-table__row {
+          margin: 0;
+          position: relative;
+          display: flex;
+          flex-wrap: wrap;
+          padding: #{15 / $base-font-size}rem;
+          height: fit-content;
+          gap: #{8 / $base-font-size}rem;
+
+          @include themify() {
+            border: 1px solid getThemifyVariable("modal-border-color");
+            background-color: getThemifyVariable("search-background-color") !important;
+          }
+
+          > th {
+            padding-left: 0;
+            width: 100%;
+            font-weight: bold;
+            margin-bottom: #{8 / $base-font-size}rem;
+          }
+
+          > td {
+            padding-left: 0;
+            width: 30%;
+          }
```

Comment:
at some point we will want to make the dates be a smaller font size and a lighter color.

Reply:
yeah will do that